### DISCUSSION TITLE
General updates and looking for usage!

### DISCUSSION THREAD

**Discussion by aarnphm** [aarnphm/whispercpp#116] [Announcements]
Hi all, Sorry for the late response from issues. Life has been very busy for me.

A little update on my side: I've been working on LLM-related topics nowadays, and currently don't have a lot of bandwidth at the moment to attend to all of the needs.

I'm writing this discussion wanting to learn more about the usage and interests of this Python binding, as well as some of the usage users currently have with the main branch.

If you wish to help me maintain the project, feel free to shoot me an email at aarnphm@bentoml.com

**Comment by AIWintermuteAI** [comment]
Hi, @aarnphm and thank you for your work!
I shot you an email about helping you to maintain the project! My fork is https://github.com/AIWintermuteAI/whispercpp/tree/develop - it has multiple issues fixed.
